collector.js

```diff
@@ -101,11 +105,11 @@ class AlAzureCollector {
         var stats = this._collectionStats;
         
         if (messages && messages.length > 0) {
-            alcollector.AlLog.buildPayload(this._hostId, this._sourceId, hm, messages, formatFun, function(err, payload){
+            alcollector.AlLog.buildPayload({hostId: this._hostId, sourceId: this._sourceId, hostmetaElems: hm, content: messages, parseCallback: formatFun, filterJson: this._filterJson, filterRegexp: this._filterRegex}, function(err, data){
                 if (err) {
                     return callback(err);
                 } else {
-                    ingestc.sendAicspmsgs(payload)
+                    ingestc.sendLogmsgs(data.payload)
```

Comment:
please also add `lmcstats` send like, https://github.com/alertlogic/al-aws-collector-js/blob/master/al_aws_collector.js#L740-L743

Reply:
And please remove `stats.putLogStats()` below

Reply:
Done